ui/src/components/Navigation.tsx

```diff
@@ -68,11 +69,10 @@ const Navigation: FC<Props> = ({ username, logout }) => {
                 nonInteractive: true,
               },
               <Button
+                element="a"
```

Comment:
Is there a specific reason why we make it an anchor element?

Reply:
Yep! We need to visit the logout URL for it to perform the logout.